test/integration/http_integration.h

```diff
@@ -26,7 +26,8 @@ class IntegrationCodecClient : public Http::CodecClientProd {
                          Upstream::HostDescriptionConstSharedPtr host_description,
                          Http::CodecClient::Type type);
 
-  IntegrationStreamDecoderPtr makeHeaderOnlyRequest(const Http::RequestHeaderMap& headers);
+  IntegrationStreamDecoderPtr makeHeaderOnlyRequest(const Http::RequestHeaderMap& headers,
```

Comment:
header only request implies end stream is true.

I think you can change these calls to startRequest if you want headers without a FIN bit.

Reply:
Right, I did not see that method.